source/extensions/omni.isaac.lab/omni/isaac/lab/managers/observation_manager.py

```diff
@@ -253,7 +253,7 @@ def compute_group(self, group_name: str) -> torch.Tensor | dict[str, torch.Tenso
                 obs = term_cfg.noise.func(obs, term_cfg.noise)
             if term_cfg.clip:
                 obs = obs.clip_(min=term_cfg.clip[0], max=term_cfg.clip[1])
-            if term_cfg.scale:
+            if term_cfg.scale is not None:
```

Comment:
These are equivalent:

```suggestion
            if term_cfg.scale:
```

Reply:
we cannot do that as it can be a tensor with multiple elements and then you get the error:

```
RuntimeError: Boolean value of Tensor with more than one value is ambiguous
```